utils/bitfield/src/lib.rs

```diff
@@ -16,6 +16,12 @@ use std::{
 
 type Result<T> = std::result::Result<T, &'static str>;
 
+// MaxEncodedSize is the maximum encoded size of a bitfield. When expanded into
+// a slice of runs, a bitfield of this size should not exceed 2MiB of memory.
+//
+// This bitfield can fit at least 3072 sparse elements.
+const MAX_ENCODED_SIZE: usize = 32 << 10;
```

Comment:
why not `1 << 15`? i assume there is a reason, just curious!

Reply:
This is a copy pasta of the way `go-bitfield` define the constant.
https://github.com/filecoin-project/go-bitfield/commit/3478657c3ba2c906251503f26f02997e466af5ed

I guess they write it like this to make `32KiB` more visible :)